app/src/main/java/co/develhope/chooseyourownbeer/BeerDetail.kt

```diff
@@ -26,10 +26,10 @@ class BeerDetail : Fragment() {
 
     }
     fun setupUI(beer: Beer) {
-        binding.imageBeer.setImageResource(beer.imagePath)
-        binding.titleBeer.text = beer.title
-        binding.size.text = beer.size.toString()
-        binding.longDescription.text = beer.fullDescription
+        binding.coloreGial.debug(beer.imagePath)
```

Comment:
Perchè sono stati cambiati i nomi?

Reply:
mi dava errore in BeerDetail.kt, cambiando i nomi il problema era risolto runnando. 
poi cambiando altro non andava piu in "run".